### PR TITLE
v0.21.1 - I've Made A Huge Mistake

### PR DESCRIPTION
**Pull request by shiftkey**
I published up v0.21 today without realising I missed calling out some of the new work that @maddin2016 had contributed. This is the remedy for that (yes, it's not technically a bugfix release but it'll help me sleep at night).

 - [x] release notes
 - [x] bump version
 - [x] :thumbsup: on release notes
 - [x] :shipit:

### CODE REVIEW

**Comment by ryangribble** [comment]
Oh dear!  I saw it mentioned 0.21 was shipping tomorrow so I didn't review it earlier (that was a task for tonight)  :grinning 

I'll check these amendments out soon

File: ReleaseNotes.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+### New in 0.21.1 (released 2016/07/29)
+
+**Features**
+
+Due to an programming error in the tool to generate these release notes, additional
```
Comment (ryangribble) [inline ReleaseNotes.md]: an programming => a programming

**Comment by ryangribble** [comment]
I've run my own "PR's merged between 2 refs" code against this release and found another couple of PRs that aren't currently in the list:

[PR#1398](https://github.com/octokit/octokit.net/pull/1398) Updated git data commit response with signature verification object via [@Sarmad93](https://github.com/Sarmad93)

[PR#1402](https://github.com/octokit/octokit.net/pull/1402) Fix serialization of enum value attributes via [@maddin2016](https://github.com/maddin2016)

[PR#1341](https://github.com/octokit/octokit.net/pull/1341) Include reactions on issue comments via [@alfhenrik](https://github.com/alfhenrik)

File: ReleaseNotes.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+features were not properly documented for the previous release:
+
+ - Reactions preview API support for issues, commit comments and PR comments - #1335, #1405 via @maddin2016
```
Comment (ryangribble) [inline ReleaseNotes.md]: There are actually 4 (not 3) items that Reactions are supported for:  commit comments, issues, issue comments, pull request review comments

**Comment by shiftkey** [comment]
@ryangribble thanks for the sanity check. I called out #1402 in the previous release, and I'll add the others here.

**Comment by ryangribble** [comment]
Weird, I'll need to look at why 1402 comes up in my query, 😀

**Comment by shiftkey** [comment]
This is shipped (again)

**Pull request merged**